### PR TITLE
add LimitRequestLine parameter

### PR DESCRIPTION
**Pull request by stefan-ahrefs**

### CODE REVIEW

**Comment by stefan-ahrefs** [comment]
ping

**Comment by daianamezdrea** [comment]
Hi, I just closed and reopened the PR in order to trigger the tests check

**Comment by stefan-ahrefs** [comment]
ping

**Comment by stefan-ahrefs** [comment]
the failed unit tests are not related to this PR

File: manifests/init.pp [hunk 1]
```diff
@@ -522,6 +522,7 @@
   $max_keepalive_requests                                               = $apache::params::max_keepalive_requests,
   $limitreqfieldsize                                                    = '8190',
   $limitreqfields                                                       = '100',
+  $limitreqline                                                         = '8190',
```
Comment (ekohl) [inline manifests/init.pp:525]: I'd like to see data types for all new parameters.

**Review by ekohl — COMMENTED** [review]
Could you also add a test for this? I'm also wondering if this should be in every config, or instead default to `undef` and only be added when it's set.

**Comment by crazymind1337** [comment]
@stefan-ahrefs can you please check the request from @ekohl. We would like to use this feature soon.

**Comment by stefan-ahrefs** [comment]
I updated yesterday. please let me know if it's good to go now or it needs further changes.

**Comment by LukasAud** [comment]
Closing and reopening the PR to rekick automated testing.

**Pull request closed without merging**